river/rule/amrules.py

```diff
@@ -95,6 +96,25 @@ def drift_test(self, y, y_pred):
         return in_drift
 
     def anomaly_score(self, x):
```

Comment:
We call this `score_one` in the `AnomalyDetector` base class :)

Reply:
I changed that! @MaxHalford, what about the main AMRules class? It does not return a single value, but a tuple. Should I also inherit from `base.AnomalyDetector`?